src/DssSpell.sol

```diff
@@ -40,52 +40,170 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", THREE_PCT_RATE, true);
+
+        // Increase the UNIV2USDCETH-A Stability Fee from 2.0% to 2.5%
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", TWO_FIVE_PCT_RATE, true);
+
+        // Increase the UNIV2UNIETH-A Stability Fee from 2.0% to 4.0%
+        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", FOUR_PCT_RATE, true);
+
+        // Decrease the GUNIV3DAIUSDC1-A Stability Fee from 0.5% to 0.1%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC1-A", ZERO_ONE_PCT_RATE, true);
+
+        // ----------------------------- Debt Ceiling updates -----------------------------
+        // Increase the WBTC-A Maximum Debt Ceiling (line) from 1.5 billion DAI to 2 billion DAI
+        // Increase the WBTC-A Target Available Debt (gap) from 60 million DAI to 80 million DAI
+        DssExecLib.setIlkAutoLineParameters("WBTC-A", 2 * BILLION, 80 * MILLION, 6 hours);
+
+        // Increase the Dust Parameter from 30,000 DAI to 40,000 DAI for the ETH-B
+        DssExecLib.setIlkMinVaultAmount("ETH-B", 40_000);
+
+        // Increase the Dust Parameter from 10,000 DAI to 15,000 DAI for all vault-types excluding ETH-B and ETH-C
+        DssExecLib.setIlkMinVaultAmount("ETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDC-B", 15_000);
+        DssExecLib.setIlkMinVaultAmount("TUSD-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-B", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-C", 15_000);
+        DssExecLib.setIlkMinVaultAmount("KNC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("MANA-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDT-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("PAXUSD-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("LINK-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("YFI-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("GUSD-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNI-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("RENBTC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("MATIC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2DAIETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2WBTCETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2USDCETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2DAIUSDC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2ETHUSDT-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2UNIETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2WBTCDAI-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2AAVEETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("UNIV2DAIUSDT-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("GUNIV3DAIUSDC1-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("GUNIV3DAIUSDC2-A", 15_000);
```

Comment:
already set in onboarding code
```suggestion

```

Reply:
@julienmartinlevrai for formality purposes (since we're dealing with 2 different polls), we're onboarding the collateral with 10,000 DAI dust first (as described in the poll) then increasing it to 15,000. Please confirm if the change is indeed necessary.